src/managing-workflow/configuring-dns.md

```diff
@@ -1,67 +1,71 @@
 # Configure DNS
 
-For Deis clusters, DNS records must be created. The Deis cluster runs multiple routers infront of the Deis controller and apps you deploy, so a [load balancer][] is recommended.
+The Deis Workflow controller and all applications deployed via Workflow are intended (by default) to be accessible as subdomains of the Workflow cluster's domain.  For example, assuming `example.com` were a cluster's domain:
 
-You can find the IP addresses of your Kubernetes cluster nodes by
-running `kubectl get nodes`.
+* The controller should be accessible at `deis.example.com`
 
-## Necessary DNS records
+* Applications should be accessible (by default) at `<application name>.example.com`
 
-Deis requires a wildcard DNS record. Assuming `myapps.com` is the top-level domain
-apps will live under:
+Given that this is the case, the primary objective in configuring DNS is that traffic for all subdomains of a cluster's domain be directed to the cluster node(s) hosting the platform's router component, which is capable of directing traffic within the cluster to the correct endpoints.
 
-* `*.myapps.com` should have "A" record entries for each of the load balancer's IP addresses
 
-Apps can then be accessed by browsers at `appname.myapps.com`, and the controller will be available to the Deis client at `deis.myapps.com`.
+## With a Load Balancer
 
-[AWS recommends][] against creating "A" record entries; instead, create a wildcard "CNAME" record entry for the load balancer's DNS name, or use Amazon [Route 53][].
+Generally, it is recommended that a [load balancer][] be used to direct inbound traffic to one or more routers.  In such a case, configuring DNS is as simple as defining a wildcard record in DNS that points to the load balancer.
 
-These records are necessary for all deployments of Deis other than Vagrant clusters.
+For example, assuming a domain of `example.com`:
 
-## DNS without a Load Balancer
+* An `A` record enumerating each of your load balancer(s) IPs (i.e. DNS round-robining)
 
```

Comment:
remove this newline? not sure if this is really here or the PR is showing the changes weird...